<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewConnectWindow</class>
 <widget class="QWidget" name="NewConnectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">QLabel{ 
font-size: 14px;
}</string>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="port_edit">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>25</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
border-radius:5px;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">QLabel{ 
font-size: 14px;
}</string>
      </property>
      <property name="text">
       <string>Connection Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="connect_name_edit">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>25</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
border-radius:5px;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">QLabel{ 
font-size: 14px;
}</string>
      </property>
      <property name="text">
       <string>Host</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="host_edit">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>25</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
border-radius:5px;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">QLabel{ 
font-size: 14px;
}</string>
      </property>
      <property name="text">
       <string>User Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="username_edit">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>25</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
border-radius:5px;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">QLabel{ 
font-size: 14px;
}</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pwd_edit">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>25</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
border-radius:5px;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>340</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="test_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Test Connection</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">QLabel{ 
font-size: 14px;
}</string>
      </property>
      <property name="text">
       <string>Database Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="db_name_edit">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>25</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
border-radius:5px;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
